src/main/java/com/olgaboiar/mint/Request.java

```diff
@@ -2,12 +2,28 @@
 
 public class Request {
     private String method;
+    private String requestedFile;
+    private String protocol;
 
-    String getMethod() {
-        return method;
+    public String getMethod() { return method; }
```

Comment:
https://github.com/olgaboiar/MintHTTPServer/pull/5/files#r252362263
RE: consistent styling

Reply:
reviewed in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/fd1dcaba0d9a4b53bed7a35a8d2e6ed75d8ee781